seacrowd/sea_datasets/duolingo_staple_2020/duolingo_staple_2020.py

```diff
@@ -0,0 +1,259 @@
+from seacrowd.utils.constants import SCHEMA_TO_FEATURES, TASK_TO_SCHEMA, Licenses, Tasks
+
+_CITATION = """\
+@inproceedings{mayhew-etal-2020-simultaneous,
+    title = "Simultaneous Translation and Paraphrase for Language Education",
+    author = "Mayhew, Stephen  and
+        Bicknell, Klinton  and
+        Brust, Chris  and
+        McDowell, Bill  and
+        Monroe, Will  and
+        Settles, Burr",
+    editor = "Birch, Alexandra  and
+        Finch, Andrew  and
+        Hayashi, Hiroaki  and
+        Heafield, Kenneth  and
+        Junczys-Dowmunt, Marcin  and
+        Konstas, Ioannis  and
+        Li, Xian  and
+        Neubig, Graham  and
+        Oda, Yusuke",
+    booktitle = "Proceedings of the Fourth Workshop on Neural Generation and Translation",
+    month = jul,
+    year = "2020",
+    address = "Online",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2020.ngt-1.28",
+    doi = "10.18653/v1/2020.ngt-1.28",
+    pages = "232--243",
+    abstract = "We present the task of Simultaneous Translation and Paraphrasing
+    for Language Education (STAPLE). Given a prompt in one language, the goal is
+    to generate a diverse set of correct translations that language learners are
+    likely to produce. This is motivated by the need to create and maintain
+    large, high-quality sets of acceptable translations for exercises in a
+    language-learning application, and synthesizes work spanning machine
+    translation, MT evaluation, automatic paraphrasing, and language education
+    technology. We developed a novel corpus with unique properties for five
+    languages (Hungarian, Japanese, Korean, Portuguese, and Vietnamese), and
+    report on the results of a shared task challenge which attracted 20 teams to
+    solve the task. In our meta-analysis, we focus on three aspects of the
+    resulting systems: external training corpus selection, model architecture
+    and training decisions, and decoding and filtering strategies. We find that
+    strong systems start with a large amount of generic training data, and then
+    fine-tune with in-domain data, sampled according to our provided learner
+    response frequencies.",
+}
+"""
+
+_DATASETNAME = "duolingo_staple_2020"
+
+_DESCRIPTION = """\
+This dataset is provided by Duolingo for their Simultaneous Translation and
+Paraphrase for Language Education (STAPLE) shared task in 2020. It contains
+English prompts and corresponding sets of plausible translations in five other
+languages, including Vietnamese. Each prompt is provided with a baseline
+automatic reference translation from Amazon, as well as some accepted
+translations with corresponding user response rates used for task scoring.
+"""
+
+_HOMEPAGE = "https://sharedtask.duolingo.com/#data"
+
+_LANGUAGES = ["vie"]
+
+_LICENSE = Licenses.CC_BY_NC_4_0.value
+
+_LOCAL = True  # needs to fill a form to download the dataset (dynamic link)
```

Comment:
mind adding the request URL here?
https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/38OJR6&version=6.0&selectTab=termsTab